graphql/resolvers.go

```diff
@@ -254,6 +254,46 @@ func (r *queryResolver) TaskTests(ctx context.Context, taskID string, sortCatego
 	return testPointers, nil
 }
 
+func setScheduled(ctx context.Context, sc data.Connector, taskID string, isActive bool) (*restModel.APITask, error) {
```

Comment:
in future it might be helpful to put functions like these in a util folder since this file is already growing pretty big